pkg/keb/cluster.go

```diff
@@ -0,0 +1,31 @@
+package keb
```

Comment:
At now, we can put this file under some test folder like `keb/test` so it can be in `test` to indicate it's only for testing purposes. 

We should not mix code for test with production together. 

And based on our AD meeting, later we will have different test layout handle this kind of code only for testing, it can be put under dedicated test directory.

https://wiki.wdf.sap.corp/wiki/display/kyma/Project+layout+for+test+code